examples/alarm.rs

```diff
@@ -1,12 +1,12 @@
-use chrono::*;
+use chrono::{Utc, Duration as ChronoDuration, };
```

Comment:
maybe instady of renaming `chrono::Duration` into "ChronoDuration" we could just import it as `chrono::Duration` instead?

Reply:
Yeah that would be better. Will change it.